content/07.prior-knowledge.md

```diff
@@ -1,14 +1,23 @@
 KGs may include links or nodes that are specific to the rare disease of interest (e.g., an FDA approved treatment  would be a specific disease-compound link in the KG) as well as links that are more generalized (e.g., gene-gene interactions noted in the literature for a different disease). 
 
-Rare disease researchers can leverage the entities and relationships in a knowledge graph outside of the specific disease-context [@doi:10.1101/727925]. 
-Such approaches have been used in rare disease research in areas such as drug repurposing [@doi:10.1101/727925] and disease classification [@doi:10.1186/s12911-019-0938-1]. 
-Identifying KG encoding methods that can provide actionable insights for a specific rare disease application is an active area of research.
+Rare disease researchers can repurpose general (i.e. not-rare-disease specific) biological or chemical knowledge graphs to answer rare disease-based research questions [@doi:10.1142/9789811215636_0041]. 
+There are a varity of tactics to sift through the large amounts of complex data in knowledge graphs.
+One such tactic is to calculate the distances between nodes of interest (e.g., diseases and drugs to identify drugs for repurposing in rare disease [@doi:10.1142/9789811215636_0041]); this is often done by determining the "embeddings" (linear representations of the position and connections of a particular point in the graph) for nodes in the knowledge graph, and calculating the similarity between these embeddings. 
+Testing a variety of methods different methods to calculate node embeddings that can generate actionable insights for rare diseases is an active area of research [@doi:10.1142/9789811215636_0041], and an opportunity for continued research.
+Another application of KGs that has been previously described is to augment or refine a dataset [@doi:10.1186/s12911-019-0752-9, doi:10.1186/s12911-019-0938-1].
+For example, Li, et. al.[@doi:10.1186/s12911-019-0938-1] used a KG to identify linked terms in a medical corpus from a large number of patients, some with rare disease diagnoses .
+They were able to augment their text dataset by identifying related terms in the clinical text to map them to the same term - an example given being mapping "cancer" and "malignancy" in different patients to the same clinical concept. 
+With this augmented and improved dataset, they were able to train and test a variety of text classification algorithms to identify rare disease patients within their corpus. 
+Finally, another possible tactic for rare disease researchers is to take a knowledge graphs (or an integration of several knowledge graphs) and apply neural network-based algorithms optimized for graph data, such as a graph convolutional neural network.
+Rao and colleagues [doi:10.1186/s12920-018-0372-8] describe the construction of a KG using phenogy information (Human Phenotype Ontology) and rare disease information (Orphanet) and curated gene interaction/pathway data (Lit-BM-13, Wikipathways). 
```

Comment:
```suggestion
Rao and colleagues [@doi:10.1186/s12920-018-0372-8] describe the construction of a KG using phenogy information (Human Phenotype Ontology) and rare disease information (Orphanet) and curated gene interaction/pathway data (Lit-BM-13, Wikipathways). 
```

Reply:
I'm not sure what `phenogy` should be instead to be honest - phenotype?

Reply:
Weird! I agree that it should be phenotype.